force-app/main/schema/classes/user/UserMapper_TEST.cls

```diff
@@ -0,0 +1,179 @@
+        Id userToTestId = UTIL_UnitTestData_TEST.createNewUserForTests(System.now().getTime() + '@testerson.com').id;
+        User userToTest = [SELECT Id, Name, isActive, Email, SmallPhotoUrl FROM User WHERE Id = :userToTestId];
+
+        Test.startTest();
+        UserModel userModel = userMapperInstance.getUserModelById(userToTest.id);
+        Test.stopTest();
+
+        System.assertEquals(userToTest.Id, userModel.Id, 'User Model id should match user id.');
+        System.assertEquals(userToTest.Name, userModel.name, 'User Model name should match user name.');
+        System.assertEquals(
+            userToTest.IsActive,
+            userModel.isActive,
+            'User Model is active should match user is active.'
+        );
+        System.assertEquals(userToTest.Email, userModel.email, 'User Model email should match user email.');
+        System.assertEquals(
+            userToTest.SmallPhotoUrl,
+            userModel.smallPhotoUrl,
+            'User Model small photo url should match user small photo url.'
+        );
+    }
+
+    /**************************************************************************************************************************
+     * @description Test method to verify that getting user models like a name is functional
+     ***************************************************************************************************************************/
+    @isTest
+    private static void getUserModelsLikeName() {
+        UserMapper userMapperInstance = UserMapper.getInstance();
+        List<Id> userToTestIdList = new List<Id>();
+        userToTestIdList.add(
+            UTIL_UnitTestData_TEST.createNewUserForTests(System.now().getTime() + '@testerson.com').id
+        );
+        userToTestIdList.add(
+            UTIL_UnitTestData_TEST.createNewUserForTests(System.now().getTime() + '@testerson.com2').id
+        );
+
+        Map<Id, User> usersByIds = new Map<Id, User>(
+            [SELECT Id, Name, isActive, Email, SmallPhotoUrl FROM User WHERE Id IN :userToTestIdList]
+        );
+
+        Test.startTest();
+        List<UserModel> userModelList = userMapperInstance.getUserModelsLikeName('Smith', 1);
+        Test.stopTest();
+
+        System.assertEquals(1, userModelList.size(), 'Only 1 user model should be retrieved.');
+        System.assertNotEquals(
+            null,
+            usersByIds.get(userModelList[0].id),
+            'The user model retrieved should be within the map of those created.'
+        );
+    }
+
+    /**************************************************************************************************************************
+     * @description Test method to verify that getting user models like a name is functional
+     ***************************************************************************************************************************/
+    @isTest
+    private static void getUserModelsLikeNameNegative() {
```

Comment:
Very minor but we might want better test methods names than "Negative"

Reply:
Updated to be NoneRetrieved for these. Negative tests do have a specific meaning.